orchagent/aclorch.cpp

```diff
@@ -4083,6 +4101,42 @@ sai_status_t AclOrch::deleteDTelWatchListTables()
     return SAI_STATUS_SUCCESS;
 }
 
+void AclOrch::registerFlexCounter(const AclRule& rule)
+{
+    SWSS_LOG_ENTER();
+
+    auto ruleIdentifier = generateAclRuleIdentifierInCountersDb(rule);
+    auto counterOidStr = sai_serialize_object_id(rule.getCounterOid());
+
+    unordered_set<string> serializedCounterStatAttrs;
+    for (auto counterAttrPair: aclCounterLookup)
```

Comment:
@stepanblyschak why not to use `const auto&` ?

Reply:
Better to use.